mixin-styles-gb/block-templates/404.html

```diff
@@ -0,0 +1,35 @@
+<!-- wp:group {"className":"mxs-wrapper"} -->
+<div class="wp-block-group mxs-wrapper"><div class="wp-block-group__inner-container"><!-- wp:template-part {"slug":"header","theme":"mixin-styles-gb","tagName":"header","align":"full","className":"mxs-header mxs-header\u002d\u002dmain"} /-->
```

Comment:
Remove all `"theme":"mixin-styles-gb"` that are not nested, inner blocks, or the "template part not found" error message will show.

Reply:
When I remove `"theme": "mixin-styles-gb"` from each template part call, the header/footer disappears and is replaced with the message "Template part has been deleted or is unavailable."

Reply:
Are you using an older version of Gutenberg? Try the development version from GitHub.

Reply:
I downloaded and installed the development version and the issue still exists. Do I need to recreate the header/footer or anything?

Reply:
Delete any existing templates that are saved under appearance > templates

Reply:
All of the templates were removed from the Appearance menu as well, but I still can't get the template parts to show up.

Reply:
After moving the generated code outside of the group block, I see that it works. Based on this, I will move the code and adjust the CSS to how it looked before.